routes/actors.js

```diff
@@ -1,15 +1,16 @@
 const router = require('koa-router')({
   prefix: '/actors'
 })
-const {apiRender, apiRenderCsv, searchCriteria, version} = require('../utils')
+const { renderFormat, searchCriteria, version } = require('../utils')
```

Comment:
apiRender is not defined, see: https://circleci.com/gh/betagouv/eac-api/156 (donc ligne 105)